### DISCUSSION TITLE
3.5 Field Layout Designer: collapsing tabs

### DISCUSSION THREAD

**Discussion by timkelty** [craftcms/cms#8885] [Ideas] [answered]
While the Field Layout Designer is 💯 times better in 3.5, with a large amount of fields in a given tab, it still suffers from some of the drag/scroll,scroll,scroll,scroll/drop issues the old version did.

I think the easiest fix would be allow users to collapse a tab, which would free up the vertical space.

**Comment by brandonkelly** [comment]
Removed the bits about multi-selection as that’s already covered by #1674.

**Comment by brandonkelly** [comment, marked as answer]
This is resolved in 5.6.0 via #16374 🎉
